FirebaseAuth/Sources/Backend/FIRAuthRequestConfiguration.h

```diff
@@ -41,6 +42,10 @@ NS_ASSUME_NONNULL_BEGIN
     @brief The heartbeat logger used to add heartbeats to the corresponding request's header.
  */
 @property(nonatomic, copy, nullable) id<FIRHeartbeatLoggerProtocol> heartbeatLogger;
+/** @property appcheck
```

Comment:
nit: Can you add one line of white-space in between the previous property and this one?

Reply:
Sure!